apps/builder/app/builder/features/style-panel/style-layer-utils.ts

```diff
@@ -211,7 +213,10 @@ export const getHumanizedTextFromLayer = (
         );
 
         if (transitionProperty === undefined) {
-          throw `Transition property is missing from the layer ${JSON.stringify(layer)}`;
+          transitionProperty = {
+            type: "keyword",
+            value: "unknown",
+          };
```

Comment:
@istarkov even if the values are added from advanced tab. It is parsing them, so the only other thing i can think og fall under this is for older projects. But for older projects there was a migration that is ran when a project is loaded. Any specific case where it is failing ?

Reply:
Ruined prod today ;) Just add new transition